pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -194,11 +194,11 @@ func (h *NetworkUtilsHandler) ConfigureIpForwarding(proto iptables.Protocol) err
 }
 
 func (h *NetworkUtilsHandler) IsIpv6Enabled(interfaceName string) (bool, error) {
-	link, err := Handler.LinkByName(interfaceName)
+	link, err := h.LinkByName(interfaceName)
```

Comment:
Since we have a discussion about the other commits in this PR. This one is for sure correct and needed, it can be moved to another PR to be merged faster.

Reply:
Good point will do that.

Reply:
Done https://github.com/kubevirt/kubevirt/pull/5512